classes/class-newsmatchdonation-shortcode.php

```diff
@@ -0,0 +1,156 @@
+<?php
+/**
+ * The class for the News Match Donation Shortcode, and associated functions
+ *
+ * @package NewsMatchDonation\Shortcode
+ */
+
+/**
+ * Register a shortcode
+ */
+class NewsMatchDonation_Shortcode {
+	/**
+	 * The prefix used for this plugin's options saved in the options table
+	 *
+	 * @var string $options_prefix The prefix for this plugin's options saved in the options table
+	 */
+	protected $option_prefix = '';
+
+	/**
+	 * The constructor
+	 */
+	public function __construct() {
+		$this->option_prefix = NewsMatchDonation_Settings::$options_prefix;
+
+		add_action( 'wp_enqueue_scripts', array( $this, 'register_assets' ) );
+		add_shortcode( 'newsmatch_donation_form', array( $this, 'donation_form_shortcode' ) );
+	}
+
+	/**
+	 * Define the donation form shortcode.
+	 * <p>
```

Comment:
why html tags here?

Reply:
Those were leftover from importing it from the Rivard plugin, which in turn was left over from the VOSD plugin, which in turn was written by a different agency. I didn't see a need to change them, but I guess it does make sense to remove them.